File: templates/lang/java/src_files/SomeClassTest.java
package {{ .Group }};

import org.junit.Test;

import static org.assertj.core.api.Assertions.assertThat;

public class SomeClassTest {

  @Test
  public void placeholderTest() {
    int input = 1;
    int expectedOutput = 2;

    int output = SomeClass.increment(input);

    assertThat(output).isEqualTo(expectedOutput);
  }
}

File: templates/lang/kotlin/src_files/SomeClassTest.kt
package {{ .Group }}

import org.assertj.core.api.Assertions.assertThat
import org.junit.Test

class SomeClassTest {

  @Test
  fun placeholderTest() {
    val input = 1
    val expectedOutput = 2

    val output = SomeClass.increment(input)

    assertThat(output).isEqualTo(expectedOutput)
  }
}
